src/uwtools/config/tools.py

```diff
@@ -248,48 +282,24 @@ def _validate_depth(
     target_class = format_to_config(target_format)
     config = config_obj.data if isinstance(config_obj, Config) else config_obj
     if bad_depth(target_class.get_depth_threshold(), depth(config)):
-        raise log_and_error(
+        raise UWConfigError(
```

Comment:
This error was being printed twice via the CLI since the CLI catches and prints/logs UW exceptions. API users can catch the exception and log if they want to.